### PR TITLE
Add nfs_ganesha and gluster_hci roles

### PR DESCRIPTION
**Pull request by sac**
This is a initial commit which implements the features:
* nfs_ganesha : create cluster
* gluster_hci : setup glusterfs and hci

### CODE REVIEW

**Comment by sac** [comment]
@devyanikota can you please review this?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,89 @@
...
+| gluster_features_ganesha_masternode |    | UNDEF | One of the nodes from the Trusted Storage Pool, gluster commands will be run on this node. gluster_features_ganesha_masternode: "{{ groups['ganesha_nodes'][0] }}" - the first node of the inventory section ganesha_nodes will be used.|
+| gluster_features_ganesha_clusternodes |    | UNDEF | List of the nodes in the Trusted Storage Pool. gluster_features_ganesha_clusternodes: "{{ groups['ganesha_nodes'] }}" - The nodes listed in section ganesha_nodes in the inventory. |
+
+### Gluster Hyperconverged Interface setup related roles
+
+| Name                     |Choices| Default value         | Comments                          |
+|--------------------------|-------|-----------------------|-----------------------------------|
+| gluster_features_hci_cluster |  | UNDEF   | The cluster ip/hostnames. Can be set by gluster_hci_cluster: "{{ groups['hc-nodes'] }}",  where hc-nodes is from the inventory file.  |
+| gluster_features_hci_volumes  |  | UNDEF | This is a dictionary setting the volume information. See below for futher explanation and variables. |
```
Comment (devyanikota) [inline README.md]: nit: s/ futher/ further

File: README.md [hunk 2]
```diff
@@ -1,2 +1,89 @@
...
+| gluster_features_hci_packages   | UNDEF | | List of packages to be installed. User need not set this, picked up from defaults. |
+| gluster_features_hci_volume_options |  | UNDEF | This is not needed to be set by user, defauts are picked up. Set to override defaults. For default values see Gluster HCI documentation. |
```
Comment (devyanikota) [inline README.md]: s/ defauts/ defaults

File: roles/gluster_hci/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+gluster_hci
+===========
+
+The role gluster_hci deploys and manages the hyperconverged interface by
+integrating Red Hat Virtualizition and GlusterFS.
```
Comment (devyanikota) [inline roles/gluster_hci/README.md]: nit: s / Virtualizition/ Virtualization

File: roles/gluster_hci/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+GlusterFS 3.2 or higher
+
+Role Variables
+--------------
+
+| Name                     |Choices| Default value         | Comments                          |
+|--------------------------|-------|-----------------------|-----------------------------------|
+| gluster_features_hci_cluster |  | UNDEF   | The cluster ip/hostnames. Can be set by gluster_hci_cluster: "{{ groups['hc-nodes'] }}",  where hc-nodes is from the inventory file.  |
+| gluster_features_hci_volumes  |  | UNDEF | This is a dictionary setting the volume information. See below for futher explanation and variables. |
```
Comment (devyanikota) [inline roles/gluster_hci/README.md]: nit: s/ futher/ further

File: roles/gluster_hci/README.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+| gluster_features_hci_packages   | | UNDEF | List of packages to be installed. User need not set this, picked up from defaults. |
+| gluster_features_hci_volume_options |  | UNDEF | This is not needed to be set by user, defauts are picked up. Set to override defaults. For default values see Gluster HCI documentation. |
```
Comment (devyanikota) [inline roles/gluster_hci/README.md]: nit: s/ defauts/ defaults

File: tests/test.yml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
+  remote_user: root
+  roles:
+    - gluster-ansible-features
```
Comment (devyanikota) [inline tests/test.yml]: gluster.features instead of gluster-ansible-features ?

**Comment by sac** [comment]
Ack! Changed.

**Pull request merged**